ansible_rulebook/cli.py

```diff
@@ -167,6 +179,14 @@ def main(args: List[str] = None) -> int:
         print("Error: inventory is required")
```

Comment:
This is not related with this PR but I'm looking this line and thinking that now inventory should not be mandatory with run_job_template action.

Reply:
agree